utils/docker/docker.go

```diff
@@ -0,0 +1,33 @@
+package docker
```

Comment:
We can't, because it introduces import cycles.  I think @vishh may have thought this was an unaddressed comment -- we discussed it in slack, recording here for posterity.